### ISSUE TITLE
when updating an export, check if the same thing is being exported

### ISSUE THREAD

**Issue by eMBee** [societyserver/sTeam#8]
it is possible to do:
    export-to-git.pike /sources/ /tmp/export-test/
followed by
    export-to-git.pike /home/sTeam/ /tmp/export-test/

this results in a branch being created based on the export time, with the history of the second export being added to the history of the first.

to fix this, check if the export source is actually the same. 
if it is different, create a new branch that is not based of the old branch, or refuse exporting.

**Comment by d4r3topk** [comment]
You mean, on exporting both, the branch with the /home/sTeam/ contains the commits of /sources/ also?

**Comment by eMBee** [comment]
yes, it does now, but it should not. so we need to detect that /sources/ and /home/sTeam are not the same, and create a new independent branch
